client/src/legacy/GridField.js

```diff
@@ -176,7 +176,8 @@ $.entwine('ss', function($) {
       this.find('.sortable-header').append('<th class="main col-Actions" />');
       this.find('tbody tr').each(function () {
         var cell = $(this).find('td:last');
-        cell.attr('colspan', 2);
+        var colspan = cell.attr("colspan") ?? 1;
```

Comment:
```suggestion
        var colspan = cell.attr('colspan') ?? 1;
```
What is the `?? 1` there for? What scenarios result in there not being a `colspan` attribute?

Reply:
The colspan attribute is not set by default, I've not seen it in use other than in the case of an empty gridfield.

Reply:
Would colspan `2` always be correct when `colspan` wasn't already defined?